test/unit/yesno.spec.ts

```diff
@@ -220,6 +220,36 @@ describe('Yesno', () => {
       expect(yesno.intercepted()).to.have.lengthOf(1);
     });
 
+    it('should support out of order mocks', async () => {
```

Comment:
I'm not sure this test is really about out of order mocks. It's more about the behavior of defining mocks with `matching...respond`. 

Out of order mock test might have like 
```
yesno.mock([createMock(path /A, response A), createMock(path /B, response B), createMock(path /C, response C)]);

Then call
```
requestTestServer(/A)
requestTestServer(/C)
requestTestServer(/B)
```

and doing above it still works.

Reply:
I think the use-case we want is. 
- I have a server that makes calls A, B, C. The order of B, C calls is non-deterministic.
- I want to run `save` to capture mocks to disk. I want to run `load` to restore mocks from disk.
- I don't want my tests to fail if the order of B and C tests fail